pkgs/by-name/sp/spice-autorandr/package.nix

```diff
@@ -0,0 +1,51 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, pkg-config
+, autoconf
+, automake
+, libX11
+, libXrandr
+}:
+
+stdenv.mkDerivation rec {
+  name = "spice-autorandr";
+  version = "0.0.2";
+
+  src = fetchFromGitHub {
+    owner = "seife";
+    repo = name;
+    rev = "0f61dc9";
+    hash = "sha256-eBvzalWT3xI8+uNns0/ZyRes91ePpj0beKb8UBVqo0E=";
+  };
+
+  nativeBuildInputs = [ pkg-config ];
+  buildInputs = [ autoconf automake libX11 libXrandr ];
```

Comment:
`autoconf` and `automake` should be in `nativeBuildInputs`.

`autoconf`, `automake`, and `preConfigure` can be removed if you use `autoreconfHook`

Reply:
Thanks, I've switched to `autoreconfHook`